core/eolearn/tests/test_eodata.py

```diff
@@ -172,6 +172,41 @@ def test_get_feature(self):
 
         self.assertTrue(np.array_equal(eop_bands, bands), msg="Data numpy array not returned properly")
 
+    def test_shallow_copy(self):
+        path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../../example_data/TestEOPatch')
```

Comment:
this should be defined as a constant at the start of file. Since these are the outdated 0.9 core tests i guess there is no fixture for it yet?

Reply:
Yes, I intentionally wrote like this for now and then improved it for the other PR for `develop-v1.0`. I hope it is ok if we leave this copy-pasta here as it has already been fixed in the new version.